system-addon/lib/HighlightsFeed.jsm

```diff
@@ -141,13 +136,10 @@ this.HighlightsFeed = class HighlightsFeed {
       case at.INIT:
         this.init();
         break;
-      case at.NEW_TAB_LOAD:
-        if (this.highlightsLength < HIGHLIGHTS_MAX_LENGTH) {
-          // If we haven't filled the highlights grid yet, fetch again.
-          this.fetchHighlights(true);
-        } else if (Date.now() - this.highlightsLastUpdated >= HIGHLIGHTS_UPDATE_TIME) {
+      case at.SYSTEM_TICK:
+        if (Date.now() - this.highlightsLastUpdated >= HIGHLIGHTS_UPDATE_TIME) {
           // If the last time we refreshed the data is greater than 15 minutes, fetch again.
-          this.fetchHighlights(false);
+          this.fetchHighlights(true);
```

Comment:
We don't want to broadcast on `SYSTEM_TICK` and don't want feed update time logic.